iina/FileGroup.swift

```diff
@@ -17,7 +17,6 @@ class FileInfo: Hashable {
   var filename: String
   var ext: String
   var nameInSeries: String?
-  var characters: [Character]
```

Comment:
This change is mainly because `self.characters = [Character](self.filename.characters)` is causing too much memory copying (and somehow shows leaking in Instruments). If you guys think performance has higher priority, I'm OK with reverting this.